src/content/docs/en/reference/feature-flags/legacy-collections.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Legacy - Collections
+sidebar:
+  label: Collections (Astro v2-v4)
```

Comment:
I think this label might be a little confusing: it makes me thing this flag is available in v2–v4 NOT that this flag is available in v5 and reintroduces the v2–v4 behaviour.

Reply:
Ah, OK, I was trying to figure out how to say "The collections from Astro versions 2 to 4" not with respect to when the flag was available.

I have gone through a few iterations in my mind:

Collections v1-- (the first "iteration" of collections)
Collections v2 / Collections (Astro v2) -- (the collections introduced in Astro v2)
Collections v2-v4 (the collections you used in versions of Astro 2 through 4)